src/devices/UFireIse/UFirePh.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+using UnitsNet;
+
+namespace Iot.Device.UFire
+{
+    /// <summary>
+    /// Get pH measuremens from μFire ISE (ion-selective electrode) Probe Interface
+    /// </summary>
+    public class UFirePh : UFireIse
+    {
+        private const float ProbeMvToPh = 59.2F;
+        private const float TemperatureCorrectionFactor = 0.03F;
+
+        /// <summary>
+        /// pH (Power of Hydrogen) units measurement
+        /// </summary>
+        public float Ph = 0;
+
+        /// <summary>
+        /// pOH units measurement, for the relationship between pH and pOH see https://www.chem.purdue.edu/gchelp/howtosolveit/Equilibrium/Calculating_pHandpOH.htm#pOH
+        /// </summary>
+        public float Poh => Ph >= 0 ? Math.Abs(14 - Ph) : float.NaN;
```

Comment:
you will need to remove this formula unless you find a link with MIT or other permissive license

Reply:
I think removing a link to non-permissive page is ok and remove absolute value (since Ph is positive anyway) since this formula is trivial to figure out from just reading the definition

Reply:
on the other hand from some sources seems relationship doesn't seem to always be true. it won't be for measuring i,e, acidity of soil. IMO we should remove this property at all